If etcd cluster has lost quorum, `etcdadm join` segfaults

```
2018/08/16 20:55:21 running "/opt/bin/etcdadm join https://10.105.16.46:2379" command. This might take some time..
2018/08/16 20:55:24 [certificates] Using the existing ca certificate and key.
[certificates] Generated server certificate and key.
[certificates] server serving cert is signed for DNS names [coreos-daniel-268-10-105-16-69platform9.sys] and IPs [127.0.0.1 10.105.16.69]
[certificates] Generated peer certificate and key.
[certificates] peer serving cert is signed for DNS names [coreos-daniel-268-10-105-16-69platform9.sys] and IPs [10.105.16.69]
[certificates] Generated etcdctl-etcd-client certificate and key.
[certificates] Generated apiserver-etcd-client certificate and key.
[certificates] valid certificates and keys now exist in "/etc/etcd/pki"

2018/08/16 20:55:24 2018-08-16 20:56:41.825736 I | [install] extracting etcd archive /var/cache/etcdadm/etcd/v3.3.8/etcd-v3.3.8-linux-amd64.tar.gz to /tmp/etcd425626077
2018-08-16 20:56:42.512163 I | [install] verifying etcd 3.3.8 is installed in /opt/bin/
2018-08-16 20:56:42.556884 I | [certificates] creating PKI assets
2018-08-16 20:56:42.556933 I | creating a self signed etcd CA certificate and key files
2018-08-16 20:56:43.308801 I | creating a new server certificate and key files for etcd
2018-08-16 20:56:43.690654 I | creating a new certificate and key files for etcd peering
2018-08-16 20:56:44.711776 I | creating a new client certificate for the etcdctl
2018-08-16 20:56:44.935711 I | creating a new client certificate for the apiserver calling etcd
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xb9544e]

goroutine 1 [running]:
github.com/platform9/etcdadm/cmd.glob..func5(0x12e86a0, 0xc42021ed20, 0x1, 0x1)
        /go/src/github.com/platform9/etcdadm/cmd/join.go:83 +0x57e
github.com/platform9/etcdadm/vendor/github.com/spf13/cobra.(*Command).execute(0x12e86a0, 0xc42021ecf0, 0x1, 0x1, 0x12e86a0, 0xc42021ecf0)
        /go/src/github.com/platform9/etcdadm/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/platform9/etcdadm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x12e8b60, 0xc420233f58, 0xb93faf, 0xc42012e900)
        /go/src/github.com/platform9/etcdadm/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/platform9/etcdadm/vendor/github.com/spf13/cobra.(*Command).Execute(0x12e8b60, 0xc42008c058, 0x0)
        /go/src/github.com/platform9/etcdadm/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/platform9/etcdadm/cmd.Execute()
        /go/src/github.com/platform9/etcdadm/cmd/root.go:21 +0x2d
main.main()
        /go/src/github.com/platform9/etcdadm/main.go:10 +0x20
```

--- a/cmd/join.go
+++ b/cmd/join.go
@@ -64,7 +64,7 @@ var joinCmd = &cobra.Command{
 		log.Println("[membership] Checking if this member was added")
 		client, err := etcd.ClientForEndpoint(endpoint, &etcdAdmConfig)
 		if err != nil {
-			log.Printf("[membership] Error checking membership: %v", err)
+			log.Fatalf("[membership] Error checking membership: %v", err)
 		}
 		ctx, cancel := context.WithTimeout(context.Background(), constants.DefaultEtcdRequestTimeout)
 		mresp, err := client.MemberList(ctx)

